src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java

```diff
@@ -115,13 +120,22 @@ private static String createPdfDocumentDescriptionFromCaseDataAndAcasCertificate
             + acasCertificate.getCertificateNumber();
     }
 
-    public PdfDecodedMultipartFile convertCaseDataToPdfDecodedMultipartFile(CaseData caseData)
+    public List<PdfDecodedMultipartFile> convertCaseDataToPdfDecodedMultipartFile(CaseData caseData)
         throws PdfServiceException {
-        byte[] pdfData = convertCaseToPdf(caseData);
-        return new PdfDecodedMultipartFile(pdfData,
-                                           createPdfDocumentNameFromCaseData(caseData),
-                                           PDF_FILE_TIKA_CONTENT_TYPE,
-                                           createPdfDocumentDescriptionFromCaseData(caseData));
+        List<PdfDecodedMultipartFile> files = new ArrayList<>();
+        files.add(new PdfDecodedMultipartFile(convertCaseToPdf(caseData, this.englishPdfTemplateSource),
+                                              createPdfDocumentNameFromCaseData(caseData, "English"),
+                                              PDF_FILE_TIKA_CONTENT_TYPE,
+                                              createPdfDocumentDescriptionFromCaseData(caseData)));
+
+        if(caseData.getClaimantType().getClaimantContactLanguage().equals("Welsh")) {
```

Comment:
Sorry dont want to be annoying, noticed tiny formatting:)

Reply:
done